src/app/redux/MarketReducer.js

```diff
@@ -1,40 +1,29 @@
 import {Map} from 'immutable';
-import createModule from 'redux-modules';
-
-
-export default createModule({
-    name: 'market',
-    initialState: Map({status: {}}),
-    transformations: [
-        {
-            action: 'RECEIVE_ORDERBOOK',
-            reducer: (state, action) => {
-                return state.set('orderbook', action.payload);
-            }
-        },
-        {
-            action: 'RECEIVE_TICKER',
-            reducer: (state, action) => {
-                return state.set('ticker', action.payload);
-            }
-        },
-        {
-            action: 'RECEIVE_OPEN_ORDERS',
-            reducer: (state, action) => {
-                return state.set('open_orders', action.payload);
-            }
-        },
-        {
-            action: 'RECEIVE_TRADE_HISTORY',
-            reducer: (state, action) => {
-                return state.set('history', action.payload);
-            }
-        },
-        {
-            action: 'APPEND_TRADE_HISTORY',
-            reducer: (state, action) => {
-                return state.set('history', [...action.payload, ...state.get('history')]);
-            }
-        }
-    ]
-});
+
+const defaultState = Map({status: {}});
+
+export default function reducer(state = defaultState, action) {
+    const payload = action.payload;
+
+    if (action.type === 'market/RECEIVE_ORDERBOOK') {
```

Comment:
`switch`

Reply:
good idea; this, as well as many other changes, are just following the style previously set up in the reducers before. this is a good time to refactor them, though